ci/02-oss-values.yaml

```diff
@@ -1,7 +1,7 @@
 # install the Ambassador API Gateway
 image:
-  repository: docker.io/datawire/ambassador
-  tag: 1.5.5
```

Comment:
wtf, how long has this tag been at 1.5.5?

Reply:
No idea. Pretty sure that the test is just "can we install from Helm", but it should be updated regularly.